tsuru/client/deploy.go

```diff
@@ -278,10 +289,75 @@ func (c *AppDeploy) Run(context *cmd.Context, client *cmd.Client) error {
 	return cmd.ErrAbortCommand
 }
 
-func targz(ctx *cmd.Context, destination io.Writer, filepaths ...string) error {
+func processTsuruIgnore(pattern string, dirPath ...string) ([]string, error) {
+	var paths []string
+	old, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	defer os.Chdir(old)
+	for _, dir := range dirPath {
```

Comment:
It would be better to rename this vars to `for _, dirPath := range dirPaths` as `dir` is also used inside the for body.